diff --git a/solvers/mhdEpotMultiRegionFoam/potentialFvPatchScalarField/potentialFvPatchScalarField.C b/solvers/mhdEpotMultiRegionFoam/potentialFvPatchScalarField/potentialFvPatchScalarField.C
index 5295c41..9f59d29 100644
--- a/solvers/mhdEpotMultiRegionFoam/potentialFvPatchScalarField/potentialFvPatchScalarField.C
+++ b/solvers/mhdEpotMultiRegionFoam/potentialFvPatchScalarField/potentialFvPatchScalarField.C
@@ -128,12 +128,9 @@ void potentialFvPatchScalarField::updateCoeffs()
     
     // On the boundary
     const fvPatchScalarField& nbrField =  
-    refCast<const fvPatchScalarField>
+    nbrPatch.lookupPatchField<volScalarField, scalar>
     (
-        nbrPatch.lookupPatchField<volScalarField, scalar>
-        (
-             this->internalField().name()
-        )
+        this->internalField().name()
     );
     // Swap to obtain full local values of neighbour internal field
     scalarField nbrIntFld(nbrField.patchInternalField());
